fdbserver/BlobManager.actor.cpp

```diff
@@ -1346,6 +1931,21 @@ ACTOR Future<Void> blobManager(BlobManagerInterface bmInterf,
 		fmt::print("Blob manager acquired lock at epoch {}\n", epoch);
 	}
 
+	try {
+		if (BM_DEBUG) {
```

Comment:
One nit here: since the blob container is only useful for pruning, and not for the rest of the blob manager's functionality, we probably don't want to block the main loop of BM startup on establishing this connection, especially since we're already sensitive to the latency of BM startup w.r.t. recovery time after a new BM starts.
Initializing this connection could be done asynchronously and only waited on by the places that need it (eg, pruning).

Reply:
Agreed!

Reply:
done